src/ImageSharp.Web/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -64,6 +64,8 @@ public static IImageSharpBuilder AddImageSharp(this IServiceCollection services)
 
             builder.SetCache<PhysicalFileSystemCache>();
 
+            builder.SetCacheKey<UriRelativeCacheKey>();
```

Comment:
Don't we want to use the lowercased cache key by default, as that prevents duplicate cached images and reduces the DOS attack surface?
```suggestion
            builder.SetCacheKey<UriRelativeCaseInsensitiveCacheKey>();
```

This should then also be updated in the sample project `Startup` class...